TodoApp.ConsoleApp/Services/TodoService.cs

```diff
@@ -1,77 +1,87 @@
 ﻿using System.Collections.Generic;
-using TodoApp.Library.Data;
-using TodoApp.Library.Models;
+using TodoApp.ConsoleApp.Repositories.Models;
+using TodoApp.ConsoleApp.Repositories.XMLRepository;
 using System.Linq;
 using System;
+using System.IO;
+using System.Xml;
+using System.Text;
 
 namespace TodoApp.ConsoleApp.Services
 {
     public class TodoService : ITodoService
     {
-        public IEnumerable<Task> GetAll()
-        {
-            XMLTaskReader reader = new XMLTaskReader();
-
-            IEnumerable<Task> tasks = reader.ReadTasks();
+        static string Path { get; set; }
+        TodoRepository repo = new TodoRepository(Path);
 
-            return tasks;
+        public IEnumerable<TodoModel> GetAll()
+        {
+            return repo.GetAll();
         }
 
-        public Task GetByID(Guid id)
+        public TodoModel GetByID(Guid id)
         {
-            IEnumerable<Task> tasks = GetAll();
-
-            Task wantedTask = null;
-
-            wantedTask = tasks.FirstOrDefault(task => task.ID == id);
-
-            return wantedTask;
+            return repo.GetById(id);
         }
 
-        public bool Update(Task task)
+        public bool Update(TodoModel model)
```

Comment:
We have `Update` method in our repo, so we can use it.
We don't need to call `Delete` and `Create` here.